db/controllers/articles-controller.js

```diff
@@ -7,4 +7,16 @@ exports.getArticlesById = (req, res, next) => {
         res.status(200).send({ article });
     })
     .catch(next)
-};
\ No newline at end of file
+};
+
+exports.postArticleComment = (req, res, next) => {
+    const { username, body } = req.body
+    const { article_id } = req.params
+    createArticleComment(article_id, username, body)
+    .then ((rows) => {
+        res.status(201).send({ comment: rows })
```

Comment:
Only an object supposed to be sent back instead of an array.

Reply:
I consoled logged rows and it is an object, so i think I'm misunderstanding this